Python/Python.sublime-syntax

```diff
@@ -364,6 +364,12 @@ contexts:
 ###[ COMMENTS ]###############################################################
 
   comments:
+    - match: '(# ?)(TODO|ToDo|Todo|todo)\b(.*)'
+      captures:
+        1: comment
+        2: string.todo
+        3: string.todo
```

Comment:
> I used `string.todo` to allow users to select alternative colors for this namespace. Using comment.todo might be a better option, but it will not highlight todos. This is because all existing themes do not use it, meaning it will remain as a regular comment.

This would give the wrong semantic meaning to the text. Sometimes I do text searches within specific scopes - like a word but only in strings, or only in comments. Scoping this wrongly would mess with this.

Instead of catering to existing color schemes like this, it would be better to stack scopes, like `comment.line` plus [`markup.info` or something](https://github.com/sublimehq/Packages/issues/1036)